README.md

```diff
@@ -25,8 +21,6 @@ end-to-end latency seen by the user when running the workload from a
 framework such as PyTorch. Supported runtimes:
 
 * JAX with IREE PJRT.
-* JAX and Tensorflow with XLA.
```

Comment:
@mariecwhite would you say these are still supported? I don't want to undocument things that still work.

Reply:
The code is still there and can be run so it's still supported.